langs/go/Dockerfile

```diff
@@ -13,8 +19,6 @@ COPY --from=0 /usr/local/go/pkg/linux_amd64              /usr/local/go/pkg/linux
 COPY --from=0 /usr/local/go/pkg/tool/linux_amd64/compile \
               /usr/local/go/pkg/tool/linux_amd64/link    /usr/local/go/pkg/tool/linux_amd64/
 
-COPY go /usr/bin/go
-
-ENTRYPOINT ["go"]
+ENTRYPOINT ["/usr/bin/go"]
```

Comment:
Any reason to change the `ENTRYPOINT` to be a full path? `/usr/bin` is in the default `PATH` in docker even on a scratch image with no explicit `ENV` lines:
```
$ docker run --rm --entrypoint sh codegolf/lang-go -c 'echo $PATH'
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Reply:
I don't think so.